journal/modernized/32-epistles.adoc

```diff
@@ -383,7 +402,11 @@ Amen.
 My dear and true love is with you, and to our children, and to all dear Friends,
 as if I named them.
 
-William Edmundson.Barbadoes, the 12th day of the Fifth month, 1684.
+[.signed-section-signature]
+William Edmundson
+
+[.signed-section-context-close]
+Barbados, the 12th day of the Fifth month, 1684
 
 === For all Friends who know the heavenly gift of Christ Jesus, from the apostles to the hindermost of the flock of Christ, that they neglect not the service of their day, according to the abilities and gifts received, and more especially those gifted for doctrine and government.
```

Comment:
Let's add a `[.blurb]` class above this heading.  Generally (not always, lol) if a heading is really long, like this, we add the `[.blurb]` designation, which sort of means, _yeah, this is a heading or title-like thing, but almost like a weird combo between a heading and a paragraph, so we'll call it a **blurb**_.

Reply:
Just like the one above.... ONLY BECAUSE this is long and would goofy with **bold** and **ALL CAPS** (like we use with the short letter headings), this one should be a blurb too. 👍